lib/Basics/Utf8Helper.h

```diff
@@ -125,9 +125,11 @@ class Utf8Helper {
 
   //////////////////////////////////////////////////////////////////////////////
   /// @brief get collator language
+  /// @param langType type of language. Now supports DEFAULT and ICU only
+  /// collation
   //////////////////////////////////////////////////////////////////////////////
 
-  std::string getCollatorLanguage();
+  std::string getCollatorLanguage(LanguageType langType = LanguageType::DEFAULT);
```

Comment:
As we now return full locale name from this method, maybe it is better to rename it to `getCollactorLocale` and corresponding `setCollatorLanguage` -> `setCollatorLocale`